metadata/demographic/unpopulated_areas_2010_approx.md

```diff
@@ -0,0 +1,68 @@
+# Title
+
+Utah Unpopulated Areas 2010 Approx
+
+## ID
+
+87112b87-1e64-49e7-a291-e35e188ca250
+
+## Brief Summary
+
+Polygon dataset of unpopulated areas in Utah dissolved as a single feature.
+
+## Summary
+
+This dataset contains a polygon representing unpopulated areas in Utah as of the 2010 Census. For 2020 Census data products, please see the [Demographic Data Index](https://gis.utah.gov/products/sgid/demographic/) available from the SGID.
```

Comment:
```suggestion
This dataset contains a polygon representing unpopulated areas in Utah, derived from 2010 Census and addressing data. For 2020 Census data products, please see the [Demographic Data Index](https://gis.utah.gov/products/sgid/demographic/) available from the SGID.
```
Clarifying to note that this is a derivative product/result of our own analysis.